source/common/protobuf/utility.cc

```diff
@@ -562,6 +562,17 @@ void MessageUtil::unpackTo(const ProtobufWkt::Any& any_message, Protobuf::Messag
   }
 }
 
+absl::Status MessageUtil::unpackToNoThrow(const ProtobufWkt::Any& any_message,
+                                          Protobuf::Message& message) {
+  if (!any_message.UnpackTo(&message)) {
+    return absl::InternalError(absl::StrCat("Unable to unpack as " +
```

Comment:
Concatenating strings with + is very inefficient. If replace `+` with `,` the StrCat will make it a lot better.

Reply:
oops, I used strcat but forgot to change the `+`.
Thanks! Done.